components/icons/PurpleIcon.tsx

```
@@ -0,0 +1,12 @@
+import React from "react";
+
+import { cn } from "@/lib/utils";
+import { type PurpleIcon } from "@/types";
+
+const PurpleIcon = ({ className, children }: PurpleIcon) => {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'PurpleIcon'. Consider to delete it or rename it.

'PurpleIcon' is defined here:


(lint/suspicious/noRedeclare)